pkgs/servers/nextcloud/default.nix

```
@@ -28,6 +28,26 @@ let
       inherit packages;
     };
 
+    outputs = [ "out" "compressed" ];
+
+    nativeBuildInputs = [ zopfli brotli ];
+
+    buildPhase = ''
+      # Create missing static gzip and brotli files
```

They are not "missing" 

```suggestion
      # Create static compressed gzip and brotli files for reverse proxies like nginx
```